src/Config/ObjectModel/ExtendedLogLevel.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Azure.DataApiBuilder.Config.ObjectModel;
 
-public enum Level
+public enum ExtendedLogLevel
```

Comment:
how is this different from: https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=net-8.0